var/spack/repos/builtin/packages/prod-util/package.py

```diff
@@ -14,9 +14,16 @@ class ProdUtil(CMakePackage):
 
     homepage = "https://github.com/NOAA-EMC/NCEPLIBS-prod_util"
     url = "https://github.com/NOAA-EMC/NCEPLIBS-prod_util/archive/refs/tags/v1.2.2.tar.gz"
+    git = "https://github.com/NOAA-EMC/NCEPLIBS-prod_util"
 
     maintainers("AlexanderRichert-NOAA", "Hang-Lei-NOAA", "edwardhartnett")
 
-    version("1.2.2", sha256="c51b903ea5a046cb9b545b5c04fd28647c58b4ab6182e61710f0287846350ef8")
```

Comment:
The general guidance (https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions) is to deprecate older versions before removing them thought there are exceptions.  Let me know if you want to deprecate or really remove this.

Reply:
Good point, I'm deprecating but keeping the older version and updating deps accordingly.